api/api.go

```diff
@@ -45,54 +43,14 @@ type ApiWrapper struct {
 
 // DeregisterOrganization is the api implementation of removing an organization from the registry
 func (apiResource ApiWrapper) DeregisterOrganization(ctx echo.Context, id string) error {
```

Comment:
Why not remove it from the interface altogether?

Reply:
Removed it